import_export/formats/base_formats.py

```diff
@@ -145,6 +147,13 @@ class HTML(TextFormat):
     TABLIB_MODULE = 'tablib.formats._html'
     CONTENT_TYPE = 'text/html'
 
+    def export_data(self, dataset, **kwargs):
```

Comment:
We might have to think about the placement of this.  The issue could potentially apply to CSV exports as well.  Maybe there are cases where exported csv gets rendered in web pages, IDK.  Perhaps this function is better placed in resources.py in one of the export methods.

Reply:
The issue I see is that "escaping" might be implementation specific - e.g. you would sanitize HTML differently than you would sanitize TOML or YAML. 

CSV, the example you given, by definition shouldn't be escaped in any way I believe, although there might be different variants on CSV as it isn't really standarized.

I think it can be made generic by standarizing the `export_data` method. I will update the PR and show you my approach so we can discuss further.